templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/versions_of_statement.html.twig

```diff
@@ -26,9 +26,11 @@
 
         {% if draftStatementVersions is defined and  draftStatementVersions|length > 0 %}
 
-            {% for index, statement in draftStatementVersions if index < lastIndex %}
-                <h2>{{ "date"|trans }}: {{ statement.versionDate|dplanDate('d.m.Y | H:i') }}</h2>
-                {{ statement.text|wysiwyg }}
+            {% for index, statement in draftStatementVersions%}
+                {% if index < lastIndex  %}
```

Comment:
you can do this more concise with the twig filter like for example in c214fe94

Reply:
Can you please send a link? Token is not clickable.

Reply:
Yes, sorry #2033

Reply:
done in [5f79a5f](https://github.com/demos-europe/demosplan-core/pull/2152/commits/5f79a5f388775759f295fe310d31312f4b32d7bd)